Userland/xargs.cpp

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2020, The SerenityOS developers.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include <AK/StdLibExtras.h>
+#include <AK/StringBuilder.h>
+#include <LibCore/ArgsParser.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/wait.h>
+
+int main(int argc, char** argv)
+{
+    if (pledge("stdio rpath proc exec", nullptr) < 0) {
+        perror("pledge");
+        return 1;
+    }
+
+    const char* command = nullptr;
+    const char* replace_str = "{}";
+    bool split_with_nulls = false;
+    const char* specified_delimiter = "\n";
+    Vector<const char*> arguments;
+    bool verbose = false;
+    const char* file_to_read = "-";
+
+    Core::ArgsParser args_parser;
+    args_parser.add_positional_argument(command, "Command to run", "command");
+    args_parser.add_positional_argument(arguments, "Initial arguments to pass to command", "initial_args", Core::ArgsParser::Required::No);
+    args_parser.add_option(replace_str, "Value to be replaced in arguments", "replace", 'I', "replace_str");
```

Comment:
I think the word you're looking for is "placeholder"

Reply:
Hmmm...sure, it's better than "replacement string", I guess